MichaelEWalker87/hogwarts-house-cup
# Rules

# Rules: 

## What's this PR do? 

* Sets the mock data file 
* Builds out the rules component 
* Connects some of the gameplay files that were needed for the rules section 
* Displays message upon submitting a rule 



## Where should the reviewer start? 

* Start with the mock files then go to the Rules.js 

## How should this be manually tested? 

* When opening the page and going to the rules section open the console and look up the mock data questions add to the array notice the message will change update the console.log and see the value has also been updated. 

## Any background context you want to provide? 

* I decided to not make the data last past refresh by choice 

## What are the relevant tickets? 
* Addresses:#22 #18
* Closes:


## Screenshots (if appropriate) 

none at this time 

## Questions:

I still need to figure out how to build out each player card
